src/Kernel/BlockClosure.class.st

```diff
@@ -845,6 +845,14 @@ BlockClosure >> valueUnpreemptively [
 	^result
 ]
 
+{ #category : #scheduling }
+BlockClosure >> valueUnwindInContext: ctxt [ 
+	| ensureContext |
+	ensureContext := self asContextWithSender: ctxt.
+	ensureContext runUntilErrorOrReturnFrom: ensureContext
+
+]
```

Comment:
For this kind of methods that can be obscure to people that do not know the internals of Block management/context/.. it might be good to add a method comment. Or maybe move the one of #doTerminationFromAnotherProcess there?